src/frontend/src/sass/_form.scss

```diff
@@ -0,0 +1,3 @@
+.form-heading {
+    font-weight: 700;
+  }
```

Comment:
You do need an *empty* end of line here

Reply:
I see what you mean. I turn off the linting function as it will throw an error.  add back